app/screens/node/NodeSetup.tsx

```diff
@@ -348,14 +345,8 @@ const NodeSetup = ({ history, location }: Props) => {
 
   return (
     <Wrapper>
-      {!hideSmesherLeftPanel && (
-        <StepsContainer
-          steps={['SETUP WALLET', 'SETUP PROOF OF SPACE']}
-          currentStep={1}
-        />
-      )}
       <CorneredContainer
-        width={!hideSmesherLeftPanel ? 650 : 760}
+        width={650}
```

Comment:
Since you've removed the sidebar at all — let's use the "whole" width instead.
Or there is any reason to keep 650?
```suggestion
        width={760}
```